dist.ini

```diff
@@ -86,3 +90,6 @@ critic_config = xt/release/perlcritic.rc
 [PodSyntaxTests]
 
 [Test::Kwalitee]
+[Test::EOL]
```

Comment:
checks for whitespace at EOL. not that this author test is failing in your code as there is lots of dead whitespace. just a nice to have